src/containers/App.js

```diff
@@ -0,0 +1,38 @@
+import React, { Component } from 'react';
```

Comment:
@cisacke so this file is the gateway to the DOM when running the dev server. it seems we have just moved away from this completely, so I just scrapped it. although the dev server has better error messages, we don't need this file as a 'testing ground' for errors and bloating the repo. Just need to figure out how to make the styleguide give us better errors....